content/chapter-01/LabStarAsAService.md

```diff
@@ -0,0 +1,23 @@
+# Lab: IaaS, PaaS, CaaS, FaaS, and DaaS
+
+* Do you have a account with your cloud provider?
+  * If you do, login into the cloud provider console interface
+  * If you do not have an account, create one right now. It should take from five to fifting minutes to do so
```

Comment:
To create an account one would need a credit card, which participant may not be willing to provide.
We will have to rely on instructor's account with individual logins created for each participant.
Organization account may or may not provide the needed level of access either.
`fifting` does not seem to appear in English Dictionary

Reply:
Changed to use instructor providers accounts.